taskcluster/ci/config.yml

```diff
@@ -1,6 +1,7 @@
 ---
 trust-domain: "translations"
 task-priority: low
+task-deadline-after: 5 days
```

Comment:
Could a task last longer than this? Do we need to bump this limit in TC?

Reply:
Ah never-mind I see you mentioned it in the commit message. What is "much, much higher"?

Reply:
Ultimately, I think we should set task deadlines at least a month ahead. (Individual tasks won't take that long, but deadlines aren't about run time, but rather about "this task must be completed by".)